test/verify/check-storage-btrfs

```diff
@@ -554,13 +554,15 @@ class TestStorageBtrfs(storagelib.StorageCase):
         # creation of btrfs partition can take a while on TF.
         with b.wait_timeout(30):
             b.wait_visible(self.card_row("Storage", name="sda"))
+
+        # Make sure Cockpit hasn't mounted it in the secret
+        # place. This should only happen in Anaconda mode.
+        self.assertNotIn("/var/lib/cockpit/btrfs", m.execute("findmnt"))
```

Comment:
AFAIUI, the code above now uses a random tmp dir instead of this static path. Could this avoid hardcoding the path at all, and instead do something like `mount | grep "^{disk}"` (to include partitions like /dev/sda5, although I suppose these aren't being used here), or the equivalent findmnt?

Reply:
Yes, good idea.

Reply:
Done.